### PR TITLE
[Term Entry] JavaScript Objects .getOwnPropertyNames() added

### LINKED ISSUE #4061: [Term Entry] JavaScript Objects .getOwnPropertyNames()
### Reason/inspiration (optional)

We would like a new term entry on the `Object.getOwnPropertyNames()` method for [Objects](https://www.codecademy.com/resources/docs/javascript/objects): getOwnPropertyNames. The entry should go in a new file under `docs/content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```js `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by antriksh-9**
### Description
I was assigned this [issue](https://github.com/Codecademy/docs/issues/4061).
Added entry for JavaScript terms: Objects .getOwnPropertyNames()
Closes #4061.

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md:16]:
```suggestion
The **`obj.getOwnPropertyNames()`** method returns all properties that are present in a given object `obj` except for those symbol-based non-enumerable properties.
```
We referring to this object as `obj` in the sentence so let's change it to that,

File: content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md:29]:
```suggestion
In this example, **`Object.getOwnPropertyNames()`** returns an array containing the property names of the `person` object:
```

File: content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md:50]:
```suggestion
In this example, the `obj` object has both enumerable and non-enumerable properties. **`Object.getOwnPropertyNames()`** includes both types of properties in the resulting array:
```

File: content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md:71]: The output coming for me has a space after and before `[]` like this :
```shell
[ 'nonEnumerableProp', 'enumerableProp' ]
```
Is it same for you too? If yes then let's add it in the output here too and the output for the above example.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have few comments for you to work on @antriksh-9

File: content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md
Comment (antriksh-9) [reply content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md:71]: No it is not same for me. 
I am not getting those spaces in the output.
Thanks.

**Comment by antriksh-9** [comment]
Hey @dakshdeepHERE I have made the necessary changes.

File: content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md
Comment (dakshdeepHERE) [reply content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md:71]: Got it @antriksh-9 then let's keep it like that only.

**Review by dakshdeepHERE — APPROVED** [review]
LGTM for second review 🚀

File: content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md
Comment (SSwiniarski) [inline content/javascript/concepts/objects/terms/getOwnPropertyNames/getOwnPropertyNames.md:16]:
```suggestion
The **`.getOwnPropertyNames()`** method returns all properties that are present in a given object except for those symbol-based non-enumerable properties.

```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@antriksh-9 I did a second review and had one comment.

**Comment by antriksh-9** [comment]
Hey @SSwiniarski I have made the necessary change.

**Review by SSwiniarski — APPROVED** [review]
@antriksh-9 Looks good.

**Comment by SSwiniarski** [comment]
@antriksh-9 you'll need to fix the lint checks in your branch before we merge. I don't have the permissions on your repo to do that.

**Pull request merged**
